metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/meter/os/cpu/AbstractCpuUsage.java

```diff
@@ -68,36 +62,21 @@ public void setUsage(double usage) {
     this.usage = usage;
   }
 
-  public void setPeriodTotalTime(long periodTotalTime) {
-    this.periodTotalTime = periodTotalTime;
-  }
-
-  protected String[] readAndSplitStat() throws IOException {
-    File file = new File(filePath);
-    String stat = FileUtils.readLines(file, StandardCharsets.UTF_8).get(0);
-    return stat.trim().split("\\s+");
-  }
-
-  public void update() {
-    String[] stats;
+  @SuppressWarnings("deprecation")
```

Comment:
Do not use deprecated method, we can write a new one if no other available

Reply:
ok